src/Microsoft.Identity.Web.Diagnostics/IdHelper.cs

```diff
@@ -34,9 +36,14 @@ internal static class IdHelper
                return version[1];
            });
 
+        public static string GetIdWebVersion()
+        {
+            return s_idWebVersion.Value;
+        }
+
         public static string CreateTelemetryInfo()
```

Comment:
Why does it need to be public?

Reply:
It's a public method in an internal class.

Reply:
I can change them.

Reply:
was this changed?